views/macros/image.html

```diff
@@ -4,9 +4,9 @@
       <img class="o-image" src="https://via.placeholder.com/300x200" alt="{{ options.alt }}">
     {% else %}
       <img class="o-image" src="{{ apos.attachments.url(imageObj, { size: options.size, crop: imageObj._crop }) }}" alt="{{ options.alt or imageObj.title }}">
-      {% if options.description === true and imageObj.description %}
+      {% if options.description === true and imageObj._description %}
```

Comment:
does `_description` ever come through? Not seeing it when logging out the object

Reply:
You might need to update (reinstall) apostrophe in your local version. It should be there.

Reply:
i see it now